wordpress-mobile/WordPress-iOS
# Reader: Recalculate cell height when Discover tab is reloaded

Fixes #21867

The new recommended tags card somehow doesn't refresh the cell's height on `reloadData`, which gets called after switching tabs. This PR fixes it with a workaround, by forcing the table view to update it's cell height by calling `beginUpdates` and followed with `endUpdates` immediately on the main queue.

## To test

- Launch the Jetpack app.
- Go to Reader > Discover tab.
- Refresh until you have a recommended tags card that spans two lines.
- Tap the Following tab.
- Tap to go back to Discover again. 
- 🔎 Verify that the recommended tags cell is displayed correctly.

## Regression Notes
1. Potential unintended areas of impact
The Discover tab will now be forced to re-render its cells' height when reloaded. 

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manually tested the changes.

3. What automated tests I added (or what prevented me from doing so)
N/A.

PR submission checklist:


UI Changes testing checklist:
